MOBILE_API.xml

```diff
@@ -932,6 +932,18 @@
         <element name="phoneNumber" since="4.0">
             <description> Optional phone number of intended location / establishment (if applicable) for SendLocation.</description>
         </element>
+        
+        <element name="timeToDestination" since="7.1">
+            <description>Optional time to destination field for navigationTexts parameter in ShowConstantTB</description>
+        </element>
+        
+        <element name="turnText" since="7.1">
+            <description>Turn text for turnList parameter of UpdateTurnList</description>
+        </element>
+        
+        <element name="navigationText" since="7.1">
+          <description>Navigation text for turnList parameter of UpdateTurnList</description>
+        </element>
```

Comment:
```suggestion
        <element name="turnText" since="7.1">
            <description>Turn text for turnList parameter of UpdateTurnList</description>
        </element>
```
It turns out that `navigationText` and `turnText` seems to reference the exact same field (`Turn.navigationText`), but it seems that `turnText` is the one used by Core, so `navigationText` should be removed since it is technically unused. Unfortunately, this will likely require a proposal revision.

Reply:
revision is proposed [here](https://github.com/smartdevicelink/sdl_evolution/pull/1109)

Reply:
I have made the necessary changes, This is ready for re-review pending acceptance of revision to the proposal